src/Sulu/Bundle/AdminBundle/Resources/js/utils/TestHelper/defaultWebspace.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+const defaultWebspace = {
+    _permissions: {},
+    allLocalizations: [],
+    customUrls: [],
+    defaultTemplates: {},
+    key: 'webspace-key',
```

Comment:
In the other helpers delivering some default values I avoided to set values like this. I tried to only use `{}`, `[]` and `undefined`, whereby undefined is always the preferred choice. This way we don't endanger ourselves of the possibility that we are coupling the tests too hard on this file and break everything when somebody wants to change this `key`. It also makes it harder to understand the tests, because you don't easily see where this value is coming from.

I would say the same is true for the `name` property. I would just leave them out and set them in every test as required.

Reply:
i set the `key` and the `name` because they are not nullable in the `Webspace` type. but omiting them in the `defaultWebspace` and setting them in the test case is okay for me too 🙂